models/game.rb

```diff
@@ -126,7 +126,14 @@ def archive!
   def to_h(include_actions: false, logged_in_user_id: nil)
     actions_h = include_actions ? actions.map(&:to_h) : []
     if !players.find { |p| p.id == logged_in_user_id } && user_id != logged_in_user_id
-      actions_h.reject! { |a| a['type'] == 'message' }
+      actions_h.map! do |a|
+        if a['type'] == 'message'
+          a['user'] = user_id
```

Comment:
why is the user being assigned here?

Reply:
Just to hide who sent the message

Reply:
ah ok, can you put a quick comment to signal that